devenv/docker/blocks/loki/README.md

```diff
@@ -0,0 +1,18 @@
+By default this block is setup to scrape logs from /var/logs. If you need to log some service from the docker-compse you can add:
+```
+    # For this to work you need to install the logging driver see https://github.com/grafana/loki/tree/master/cmd/docker-driver#plugin-installation
+    logging:
+      driver: loki
+      options:
+        loki-url: "http://loki:3100/loki/api/v1/push"
+```
+
```

Comment:
I am not sure if I am doing something wrong, but I am not getting any logs with **traceId**. 
Is my config wrong? Or anything else? Also, which logs are going to have traceId? 
``` 
# datasource URL: http://localhost:3100/
  loki:
    image: grafana/loki:master
    ports:
      - "3100:3100"
    command: -config.file=/etc/loki/local-config.yaml
    logging:
      driver: loki
      options:
        loki-url: "http://loki:3100/loki/api/v1/push" 
    environment:
      - JAEGER_AGENT_HOST=tempo
      - JAEGER_AGENT_PORT=6831
      - JAEGER_SAMPLER_TYPE=const
      - JAEGER_SAMPLER_PARAM=1
  promtail:
    image: grafana/promtail:master
    volumes:
      - ./docker/blocks/loki/config.yaml:/etc/promtail/docker-config.yaml
      - /var/log:/var/log
      - ../data/log:/var/log/grafana
    command:
      -config.file=/etc/promtail/docker-config.yaml
```

Reply:
I think that should work but only query logs get traceId. So I have to run a query first (the same to get the logs) and then I get a log like this:
![Screenshot from 2021-03-08 11-29-10](https://user-images.githubusercontent.com/1014802/110309409-98b12180-8001-11eb-8864-38cdab02677e.png)